### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by MKletz**
- Added disclaimer about composite resource support in various platforms
- Added support statement
- Added missing example for creating a browser benchmark
- Added design choices document to explain some of the less obvious decisions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,4 +44,9 @@ You can be notified of new releases by following the [notifications documentatio
 The process of customizing these resources for your environment is outlined in [customization](docs/customization.md).
 
 # How can I contribute?
-Contribution information can be found in the [contributions documentation](docs/contribution.md).
\ No newline at end of file
+Contribution information can be found in the [contributions documentation](docs/contribution.md).
+
+# What is the support lifecycle for resources?
+- Resources will be updated whenever CIS publishes a new draft of a given resource.
```
Comment (edthedev) [inline README.md:50]: Excellent addition. Great transparency here. Thanks.

File: docs/design_choices.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Design choices
```
Comment (edthedev) [inline docs/design_choices.md:1]: This document is a great addition. Thanks.

File: docs/new_resources.md [hunk 3]
```diff
@@ -49,6 +49,22 @@ $Splat = @{
 }
 ConvertTo-DSC @Splat
 ```
+
```
Comment (edthedev) [inline docs/new_resources.md:52]: Can we add 

## Microsoft Edge Example

here to clarify?

File: docs/new_resources.md [hunk 4]
```diff
@@ -33,7 +33,7 @@ Set-Location -Path '.\SecOps-Powershell-CISDSC'
 .\tools\install_dependencies.ps1
 ```
 
-3) The following example will generate a new DSC composite resource. You will need to edit the parameters accordingly.
+3) The following examples will generate a new DSC composite resource for OS and a browser respectively. You will need to edit the parameters accordingly.
```
Comment (edthedev) [inline docs/new_resources.md:36]: Can we add 

```## Empty Resource``` 

here to clarify?

**Review by edthedev — CHANGES_REQUESTED** [review]
This is excellent.

Can we add a couple of headers to the examples for more context?

File: docs/new_resources.md (hunk 3 above)
Comment (MKletz) [reply docs/new_resources.md:53]: Can you write that as a suggestion block? Not sure I understand

Comment (MKletz) [reply docs/new_resources.md:53]: Did the latest commit solve this?

File: docs/new_resources.md (hunk 4 above)
Comment (MKletz) [reply docs/new_resources.md:36]: Did the latest commit solve this?

File: docs/new_resources.md (hunk 3 above)
Comment (edthedev) [reply docs/new_resources.md:53]: Yep. Thanks!

File: docs/new_resources.md (hunk 4 above)
Comment (edthedev) [reply docs/new_resources.md:36]: Yes. Thanks!

**Review by edthedev — APPROVED** [review]
This is excellent. Thanks!

**Review by tzturner — APPROVED** [review]

**Pull request merged**